joomla/joomla-cms
# Update jQuery autocomplete to 1.2.18

### How to test
1. Enable the Smart Search (mod_finder) module
2. Ensure the **Search Suggestions** is set to **Show**
3. In the frontend, type a word in the search input field
### Expected result

A dropdown will appear with associated results as shown in the image below

![search](https://cloud.githubusercontent.com/assets/2019801/6846186/00b92378-d3b4-11e4-88ee-8762eeacde02.jpg)
